tests/json-fixtures/test_blockchain.py

```diff
@@ -116,7 +115,14 @@ def test_blockchain_fixtures(fixture_name, fixture):
     #     assert rlp.encode(genesis_header) == fixture['genesisRLP']
 
     db = MemoryDB()
-    evm = MainnetEVM.from_genesis(
+
+    evm = MainnetEVM
+    if fixture_name.startswith('Homestead'):
+        evm = EVM.configure(
+            'HomesteadEVM',
+            vm_configuration=[(0, HomesteadVM)])
```

Comment:
Yeah, I agree the current approach is not the most elegant, but if this turns out to be the only place where we'd use it, it may not be worth the effort. I'll add a TODO to make sure we re-evaluate the possibility of doing what you suggest later, if that's ok with you.